src/pages/get-help/index.tsx

```diff
@@ -7,8 +7,9 @@ const GetHelp = () => {
 
   return (
     <div className="container my-5">
-      <Accordion data={data[0]} />
-      <Accordion data={data[1]} />
+      {data.map((item, index) => (
+        <Accordion key={index} data={item} />
+      ))}
```

Comment:
Putting the key like you did here is fine. Another thing that is not recommended is to use the index as a key, the reason is in the documentation, I think we could use the title instead.

Reply:
I think I fixed it. Can you read it?

Reply:
Everything is alright. I'll approve your changes.